### PR TITLE
Add information on running local cluster

### PR DESCRIPTION
**Pull request by crimsonfaith91**
- also include tip to check logs

### CODE REVIEW

File: contributors/devel/development.md [hunk 1]
```diff
@@ -117,6 +117,9 @@ Kubernetes maintains state in [`etcd`][etcd-latest], a distributed key store.
 
 Please [install it locally][etcd-install] to run local integration tests.
 
+### Development using local cluster
+After installing etcd, we recommend running a local cluster for development purpose. Refer this [tutorial](https://kubernetes-v1-4.github.io/docs/getting-started-guides/locally/) for more information.
```
Comment (MHBauer) [inline contributors/devel/development.md]: Format in shorter lines, please.

That doesn't seem like a very helpful link to me.

It immediately says, stop, go use minikube, and minikube cannot be used for development so, it is deceptive, if anything.

Possibly link to 
https://kubernetes-v1-4.github.io/docs/getting-started-guides/locally/#starting-the-cluster
or 
https://kubernetes-v1-4.github.io/docs/getting-started-guides/locally/#i-changed-kubernetes-code-how-do-i-run-it

That whole document should probably go away and and any mention of `hack/local-up-cluster.sh` should be mentioned inline.

**Comment by crimsonfaith91** [comment]
@MHBauer Both local cluster and minikube bring up one-node cluster. The local cluster is the suggested way to start a development cluster, while the minikube is more for "trying out" a stable release rather than for development.

I am following up with cluster lifecycle team about the best way to set up a cluster with multiple nodes. You can check the update here: https://groups.google.com/forum/#!topic/kubernetes-sig-cluster-lifecycle/Zp3Y4uNHnPQ

**Comment by crimsonfaith91** [comment]
@foxish are we going to merge this?

**Comment by crimsonfaith91** [comment]
@MHBauer Will this be fine for merging?

**Comment by crimsonfaith91** [comment]
i will close this due to no reply. feel free to open it again with feedback, and i will continue working on it. :)
thanks!

**Pull request closed without merging**